pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -258,6 +258,13 @@ func (is *InstanceService) InstanceCreate(clusterName string, name string, confi
 	if config == nil {
 		return nil, fmt.Errorf("create Options need be specified to create instace")
 	}
+	trunkSupport, err := GetTrunkSupport(is)
+	if err != nil {
+		return nil, fmt.Errorf("Obtaining network extensions err: %v", err)
```

Comment:
Do we have to return an error in this case?

What about something like this:

```go
if (err != nil && config.Trunk == true)  || (trunkSupport == false && config.Trunk == true) {
     return nil, fmt.Errorf("There was an issue verifying whether trunk support is available, please disable it: {} {}", config.Trunk, err)
}
```

(of course I didn't test that code)

Basically, I don't think we should fail/return in cases where `config.Trunk` is set to false and there was a problem getting network extensions.

Reply:
makes sense! thanks! I'll update it asap